docscan/session/create/check/document_authenticity.go

```diff
@@ -1,28 +1,42 @@
 package check
 
 import (
+	"encoding/json"
+
 	"github.com/getyoti/yoti-go-sdk/v3/docscan/constants"
 )
 
 // RequestedDocumentAuthenticityCheck requests creation of a Document Authenticity Check
 type RequestedDocumentAuthenticityCheck struct {
-	RequestedCheck
-	Config RequestedDocumentAuthenticityConfig `json:"config"`
+	checkType string
+	config    RequestedDocumentAuthenticityConfig
 }
 
-// RequestedDocumentAuthenticityConfig is the configuration applied when creating a Document Authenticity Check
-type RequestedDocumentAuthenticityConfig struct {
-	RequestedCheckConfig
+// Type is the type of the Requested Check
+func (c RequestedDocumentAuthenticityCheck) Type() string {
+	return c.checkType
```

Comment:
```suggestion
// Type is the type of the Requested Check
func (c RequestedDocumentAuthenticityCheck) Type() string {
	return constants.IDDocumentAuthenticity
```

Reply:
Ah yes I did this so we didn't add the constant twice, but removing it from the build method, and having this line as you suggested would make more sense I think

Reply:
Fixed in 11cfa6c